wolfcrypt/src/asn.c

```diff
@@ -162,6 +162,9 @@ extern int wc_InitRsaHw(RsaKey* key);
     #include <wolfssl/wolfcrypt/cryptocb.h>
 #endif
 
+#ifdef WOLFSSL_IP_ALT_NAME
+    #include <wolfssl/ssl.h>
```

Comment:
We cannot include ssl.h from wolfCrypt. If there are missing dependencies they should be moved to asn.h. Is it just `WOLFSSL_MAX_IPSTR` missing? Please move it.